CHANGELOG.md

```diff
@@ -372,7 +376,12 @@ Serenity now uses Rust edition 2021, with an MSRV of Rust 1.74.
 * [#2569](https://github.com/serenity-rs/serenity/pull/2569) - Replaced the `json::prelude` module with public wrapper functions that abstract over both `serde_json` and `simd-json`.
 * [#2593](https://github.com/serenity-rs/serenity/pull/2593) - Rename `GatewayIntents::GUILD_BANS` to `GUILD_MODERATION` (the old name is still present but is deprecated).
 * [#2598](https://github.com/serenity-rs/serenity/pull/2598) - Change `CreateInteractionResponseMessage::flags` to take `InteractionResponseFlags` instead of `MessageFlags`.
-
+* [#2609](https://github.com/serenity-rs/serenity/pull/2609) - Split parts of `ThreadMember` into `PartialThreadMember`
+* [#2612](https://github.com/serenity-rs/serenity/pull/2612) - Make `Event::Unknown` wrap `UnknownEvent` again
+* [#2615](https://github.com/serenity-rs/serenity/pull/2615) - Rework framework builders to not use closures
+* [#2622](https://github.com/serenity-rs/serenity/pull/2622) - Implement role addition/removal using dedicated endpoints
+* [#2623](https://github.com/serenity-rs/serenity/pull/2623) - Use dedicated types for `GuildId::audit_logs`
+* [#2625](https://github.com/serenity-rs/serenity/pull/2625) - Remove some miscellaneous vectors
```

Comment:
```suggestion
* [#2609](https://github.com/serenity-rs/serenity/pull/2609) - Split parts of `ThreadMember` into `PartialThreadMember`.
* [#2622](https://github.com/serenity-rs/serenity/pull/2622) - Implement role addition/removal using dedicated endpoints.
* [#2623](https://github.com/serenity-rs/serenity/pull/2623) - Use a dedicated type for the `action_type` parameter of `GuildId::audit_logs`.
* [#2625](https://github.com/serenity-rs/serenity/pull/2625) - Change `Guild::members_with_status` to return `impl Iterator<Item = &Member>` instead of `Vec<Member>`.
```

 - I think #2612 can be combined with the line for #1938, or just omitted entirely.
 - #2615 can instead be included in the `Framework` section of the changelog. The first sentence of that section already mentions the `configure` method, so it can be added to that.